hasura/metadata/actions.graphql

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
??? why does this need blank lines in an empty file?

Reply:
Magic auto generated stuffs